connector/src/main/java/org/jboss/as/connector/subsystems/datasources/AbstractDataSourceAdd.java

```diff
@@ -206,6 +207,8 @@ void firstRuntimeStep(OperationContext context, ModelNode operation, ModelNode m
                 dataSourceService.getDriverInjector());
 
          // If the authentication context is defined, add the capability
+         // TODO ELYTRON_ENABLED is redundant as AUTHENTICATION_CONTEXT implies elytron
```

Comment:
@maeste Thanks; I see it now. Using a SubjectFactory driven by the current context is distinct from not having a SubjectFactory at all, so getting into elytron-enabled is not relevant to the problem this JIRA is addressing. I'll remove the code comments.